Makefile

```diff
@@ -111,6 +132,7 @@ diff_pylint_report: pylint_report.txt
 # We need to get coverage to look at our scripts. Since they aren't in a
 # python module we can't tell nosetests to look for them (via an import
 # statement). So we run nose inside of coverage.
+## .coverage            : test code coverage
```

Comment:
I only rarely run this target directly; probably doesn't make sense to advertise